### DISCUSSION TITLE
Document loading failed - Failed to load Nextcloud Office - please try again later

### DISCUSSION THREAD

**Discussion by nn0de** [nextcloud/all-in-one#2337] [Questions] [answered]
As the title suggests I am having issues with Nextcloud Office on a fresh install. First time user and this is very frustrating I can't figure out what the issue is. I am getting errors in the logs of the Colabora container (attached below) 
I see there seems to be a whole heap of issues around colabora and office as of late so I couldn't quite figure out the issue as it was such a sea of code tech talk and I'm new to all this business so I haven't been able to find the fix despite trying a bunch of stuff from forums and what not. Any help would be greatly appreciated as I would love to be able to use this function to edit documents.

wsd-00001-00201 2023-04-14 22:48:37.132926 +1000 [ docbroker_009 ] ERR  Failed to add session to [MYDOMAIN:443/index.php/apps/richdocuments/wopi/files/416_ocvvrulyqapn] with URI [MYDOMAIN/index.php/apps/richdocuments/wopi/files/416_ocvvrulyqapn?access_token=IOLkp8Zm2gLQO3HMTOGY6o52HCtUadb7G&access_token_ttl=16815124163000&permission=edit]: Access denied, 403. WOPI::CheckFileInfo failed on: MYDOMAIN/index.php/apps/richdocuments/wopi/files/416_ocvvrulyqapn?access_token=IOLkp8Zm2gLQOHMTO5GY6o52HCtUadb7G&access_token_ttl=16815124623000&permission=edit| wsd/DocumentBroker.cpp:2418
wsd-00001-00201 2023-04-14 22:48:37.132983 +1000 [ docbroker_009 ] ERR  Unauthorized Request while starting session on MYDOMAIN:443/index.php/apps/richdocuments/wopi/files/416_ocvvrulyqapn for socket #22. Terminating connection. Error: Access denied, 403. WOPI::CheckFileInfo failed on: MYDOMAIN/index.php/apps/richdocuments/wopi/files/416_ocvvrulyqapn?access_token=IOLkp8Zm2gLQOHMTOGYn6o52HCtUadb7G&access_token_ttl=16815124623000&permission=edit| wsd/COOLWSD.cpp:4936
wsd-00001-00201 2023-04-14 22:48:37.193856 +1000 [ docbroker_009 ] ERR  #29: Read failed, have 0 buffered bytes (ECONNRESET: Connection reset by peer)| net/Socket.hpp:1151
wsd-00001-00201 2023-04-14 22:48:37.194002 +1000 [ docbroker_009 ] WRN  #29: Unassociated Kit (200) disconnected unexpectedly| wsd/COOLWSD.cpp:3426
frk-00046-00046 2023-04-14 22:48:37.670509 +1000 [ forkit ] WRN  No live Kits exist, and we are not terminating yet.| kit/ForKit.cpp:307
sh: 1: /usr/bin/coolmount: Operation not permitted
sh: 1: /usr/bin/coolmount: Operation not permitted
sh: 1: /usr/bin/coolmount: Operation not permitted
frk-00046-00046 2023-04-14 22:48:38.271791 +1000 [ forkit ] WRN  The systemplate directory [/opt/cool/systemplate] is read-only, and at least [/opt/cool/systemplate//etc/hosts] is out-of-date. Will have to copy sysTemplate to jails. To restore optimal performance, make sure the files in [/opt/cool/systemplate/etc] are up-to-date.| common/JailUtil.cpp:438

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?
